graphql/schema.graphql

```diff
@@ -54,6 +54,16 @@ enum SortDirection {
   DESC
 }
 
+enum MetStatus {
```

Comment:
There should be a pending status shouldn't there? In the design this would be the lack of a checkmark or X